### PR TITLE
Arrays

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
Please review and comment / commit if anything is wrong  
And title of "call array this" task needs a little more attention

### CODE REVIEW

File: 1-js/05-data-types/04-array/10-maximal-subarray/solution.md [hunk 1]
```diff
@@ -1,43 +1,43 @@
-# The slow solution
+# راه حل کند
```
Comment (mahdyar) [inline 1-js/05-data-types/04-array/10-maximal-subarray/solution.md:1]:
```suggestion
# راه حل کُند
```

File: 1-js/05-data-types/04-array/3-call-array-this/task.md [hunk 2]
```diff
@@ -2,9 +2,9 @@ importance: 5
 
 ---
 
-# Calling in an array context
+# صدازدن در محتوای یک آرایه
```
Comment (mahdyar) [inline 1-js/05-data-types/04-array/3-call-array-this/task.md:5]:
```suggestion
# فراخوانی محتوای یک آرایه
```

File: 1-js/05-data-types/04-array/3-call-array-this/solution.md [hunk 3]
```diff
@@ -11,5 +11,5 @@ arr.push(function() {
 
 arr[2](); // a,b,function(){...}
 ```
-
-The array has 3 values: initially it had two, plus the function. 
+
```
Comment (mahdyar) [inline 1-js/05-data-types/04-array/3-call-array-this/solution.md:14]: ِYou've added an extra white-space, remove it:
```suggestion

```

File: 1-js/05-data-types/04-array/5-array-input-sum/solution.md [hunk 4]
```diff
@@ -8,9 +8,9 @@ function sumInput() {
 
   while (true) {
 
-    let value = prompt("A number please?", 0);
+    let value = prompt("لطفا یک عدد وارد کنید.", 0);
 
-    // should we cancel?
+    // آیا باید متوقف کنیم؟
```
Comment (mahdyar) [inline 1-js/05-data-types/04-array/5-array-input-sum/solution.md:13]:
```suggestion
    // باید لغو کنیم؟
```

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by mahdiHashemi14** [comment]
Thanks for your review, @mahdyar
/done

**Review by mahdyar — APPROVED** [review]
Thank you, @mahdiHashemi14!

**Pull request merged**